src/PreLiquidation.sol

```diff
@@ -150,7 +150,8 @@ contract PreLiquidation is IPreLiquidation, IMorphoRepayCallback {
         require(borrowed > collateralQuoted.wMulDown(PRE_LLTV), ErrorsLib.NotPreLiquidatablePosition());
 
         uint256 ltv = borrowed.wDivUp(collateralQuoted);
-        uint256 preLIF = (ltv - PRE_LLTV).wDivDown(LLTV - PRE_LLTV).wMulDown(PRE_LIF_2 - PRE_LIF_1) + PRE_LIF_1;
+        uint256 quotient = (ltv - PRE_LLTV).wDivDown(LLTV - PRE_LLTV);
```

Comment:
feel like it's a bit clearer with quotient but no strong feeling either

Reply:
cursor is fine yes, not sure we have to go that far for a parameter that is only used to simplify computation though

Reply:
Yes not a big deal for me either

Reply:
cursor is already used in the liquidations in blue, let's try to avoid it

Reply:
> cursor is already used in the liquidations in blue, let's try to avoid it

But that's because it's the same meaning: it's a cursor between 2 extremes. So to me it's a good sign that we arrive at the same naming. I think it's fine here to have cursor because it's only a local variable. But if we want to disambiguate, then we could write something like `ltvCursor`

Reply:
I think we should keep `quotient`, it's more accurate in this case

Reply:
>But that's because it's the same meaning: it's a cursor between 2 extremes. So to me it's a good sign that we arrive at the same naming. I think it's fine here to have cursor because it's only a local variable. But if we want to disambiguate, then we could write something like ltvCursor

I get it but in Blue cursor is a parameter, while here it's more a "state variable". Not convinced..